src/tests/test_transform.py

```diff
@@ -16,88 +16,140 @@
 		addon = MockAddon()
-		addon.minNVDAVersion = V_2020_1
+		addon.minNvdaAPIVersion = V_2020_1
 		addon.lastTestedVersion = V_2020_2
-		self.assertFalse(_isAddonCompatible(addon, nvdaVersion2021_1))
+		self.assertFalse(_isAddonCompatible(addon, nvdaAPIVersion2021_1))
 
 	def test_not_valid_because_old_api(self):
 		"""Confirm an addon is not compatible with an old API"""
 		addon = MockAddon()
-		addon.minNVDAVersion = V_2021_1
+		addon.minNvdaAPIVersion = V_2021_1
 		addon.lastTestedVersion = V_2021_2
-		self.assertFalse(_isAddonCompatible(addon, nvdaVersion2020_3))
+		self.assertFalse(_isAddonCompatible(addon, nvdaAPIVersion2020_3))
 
 
 class Test_getLatestAddons(unittest.TestCase):
 	def test_beta_stable(self):
 		"""Ensure addons are added to the correct channels"""
-		NVDAVersions = (nvdaVersion2020_2,)
+		nvdaAPIVersions = (nvdaAPIVersion2020_2,)
 		betaAddon = MockAddon()
-		betaAddon.minNVDAVersion = V_2020_1
+		betaAddon.minNvdaAPIVersion = V_2020_1
 		betaAddon.lastTestedVersion = V_2020_2
 		betaAddon.channel = "beta"
 		betaAddon.pathToData = "beta-path"  # unique identifier for addon metadata version
 		betaAddon.addonVersion = MajorMinorPatch(0, 2, 1)
 		stableAddon = deepcopy(betaAddon)
 		stableAddon.channel = "stable"
 		stableAddon.pathToData = "stable-path"  # unique identifier for addon metadata version
-		self.assertDictEqual(getLatestAddons([betaAddon, stableAddon], NVDAVersions), {
+		self.assertDictEqual(getLatestAddons([betaAddon, stableAddon], nvdaAPIVersions), {
 			V_2020_2: {"beta": {betaAddon.addonId: betaAddon}, "stable": {stableAddon.addonId: stableAddon}},
 		})
 
 	def test_onlyNewerUsed(self):
 		"""Ensure only the newest addon is used for a version+channel"""
-		NVDAVersions = (nvdaVersion2020_2, nvdaVersion2020_3)
+		nvdaAPIVersions = (nvdaAPIVersion2020_2, nvdaAPIVersion2020_3)
 		oldAddon = MockAddon()
 		oldAddon.addonId = "foo"
-		oldAddon.minNVDAVersion = V_2020_2
+		oldAddon.minNvdaAPIVersion = V_2020_2
 		oldAddon.lastTestedVersion = V_2020_2
 		oldAddon.channel = "stable"
 		oldAddon.pathToData = "old-path"
 		oldAddon.addonVersion = MajorMinorPatch(0, 1)
+
 		newAddon = deepcopy(oldAddon)
-		newAddon.addonVersion = MajorMinorPatch(0, 2)
-		newAddon.minNVDAVersion = V_2020_3
+		newAddon.addonVersion = MajorMinorPatch(0, 3)
+		newAddon.minNvdaAPIVersion = V_2020_3
 		newAddon.lastTestedVersion = V_2020_3
 		newAddon.pathToData = "new-path"
-		self.assertDictEqual(getLatestAddons([oldAddon, newAddon], NVDAVersions), {
+
+		self.assertDictEqual(getLatestAddons([oldAddon, newAddon], nvdaAPIVersions), {
+			# When adding 1st addon: oldAddon.addonId not in addons
+			# When adding 2nd addon: pass as incompatible
 			V_2020_2: {"stable": {"foo": oldAddon}, "beta": {}},
+			# When adding 1st addon: oldAddon.addonId not in addons
+			# When adding 2nd addon: newAddon.addonVersion > oldAddon.addonVersion
 			V_2020_3: {"stable": {"foo": newAddon}, "beta": {}},
```

Comment:
I don't really understand this comment, specifically the "adding 1st/2nd addon". This only needs to be framed with reference to `getLatestAddons`, unless this is explaining some general "why" for the system?

I've just noted that this is testing that the order of the addons doesn't affect the output.
To make that clearer you could put the 'expected' into a constant, so it's easy to confirm it is the same for both cases.
A comment to draw attention to the order of the addon list would also help.

Reply:
I've updated the comments and documentation further to explain this

Reply:
Perhaps "included" would be clearer? Or can you answer: "added to what?"

Reply:
maybe "stored"? as in they are stored in the addons dictionary.

Reply:
Oh, ok now I see what you're trying to say. It would be clearer to make it a full sentence. I'd discard the "obvious" bit (old.addonId not in addons), presumably that is not the behavior you wish to draw attention to here, and is covered in other tests.

EG
```
# For V_2020_2 newAddon is incompatible
...
# For V_2020_3 oldAddon overrides newAddon because newAddon.addonVersion > oldAddon.addonVersion
```

Reply:
I've turned this into a generator to make the ordering a bit more clear/important and fixed the comments.